src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -8077,5 +8077,403 @@ .maxstack  2
 }
 ");
         }
+
+        [Fact]
+        public void ForeachStatement_Reorder()
```

Comment:
Thank you! Fixed

---
In reply to: [115576722](https://github.com/dotnet/roslyn/pull/19283#discussion_r115576722) [](ancestors = 115576722,115574092,115573899)